server/service/src/report/report_service.rs

```diff
@@ -589,9 +545,9 @@ fn transform_data(
         // Mapping to string via format_error since it's better then debug output on error
         ConvertDataType::BoaJs => transform_data_boajs(data, convert_data)
             .map_err(|e| ConvertDataError::BoaJs(format_error(&e))),
-        ConvertDataType::Extism => {
-            transform_data_extism(connection, data, convert_data).map_err(ConvertDataError::Extism)
-        }
+        ConvertDataType::Extism => Err(ConvertDataError::Extism(anyhow::anyhow!(
+            "Extism convert data no longer implemented."
+        ))),
```

Comment:
I thought would be good to leave this as is as likely that testers will encounter conflicting build / report versions.

Alternatively could remove the convert_data_type from the report-manifest.json altogether (and these types in rust)? I wasn't too sure.

Reply:
This is great